sdk/keyvault/azure-security-keyvault-common/src/keyvault_pipeline.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think we need this file at or or its .hpp file

Reply:
Would it be something we might be missing or not doing right by following the same .NET approach?
I see a lot of benefit on it.
The APIs implementation becomes much simpler